selenium_tests/lti_emailer/emailer_test.py

```diff
@@ -0,0 +1,23 @@
+__author__ = 'hic048'
+
+import unittest
+
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium_tests.lti_emailer.page_objects.emailer_mainpage import MainPage, Locator1
+
+class test_is_loaded(EmailerBaseTestCase):
```

Comment:
Usually classes are in CamelCase in Python, and our current convention is to call them e.g. SearchTests, DeleteTests, PermissionsTests, etc. (see icommons_lti_tools).